spotbugs/etc/messages_ja.xml

```diff
@@ -2829,44 +2893,44 @@ URL の <code>equals</code> と <code>hashCode</code> は、ドメイン名の
     <Details>
 <![CDATA[
 <p>
-URL の <code>equals</code> メソッドと <code>hashCode</code> メソッドは、ドメイン名の解決を行うので、ひどい性能になる可能性があります。<br>
-詳細は、<a href="http://michaelscharf.blogspot.com/2006/11/javaneturlequals-and-hashcode-make.html">http://michaelscharf.blogspot.com/2006/11/javaneturlequals-and-hashcode-make.html</a> を参照してください。<br>
+<code>URL</code> の <code>equals</code> メソッドと <code>hashCode</code> メソッドは，ドメイン名の解決を行うので，ひどい性能になる可能性があります。<br>
+詳細は， <a href="http://michaelscharf.blogspot.com/2006/11/javaneturlequals-and-hashcode-make.html">http://michaelscharf.blogspot.com/2006/11/javaneturlequals-and-hashcode-make.html</a> を参照してください。<br>
 その代わりに <code>java.net.URI</code> を使用することを検討してください。
 </p>
 ]]>
     </Details>
   </BugPattern>
 
   <BugPattern type="DMI_ANNOTATION_IS_NOT_VISIBLE_TO_REFLECTION">
-    <ShortDescription>ランタイムリテンションなしで、アノテーションの存在を調べるためにリフレクションを使用することはできない</ShortDescription>
+    <ShortDescription>ランタイムリテンションなしで，アノテーションの存在を調べるためにリフレクションを使用することはできない</ShortDescription>
     <LongDescription>ランタイムリテンションを持たないアノテーション {3} の存在を調べるためにリフレクションを使用しています。{1}</LongDescription>
     <Details>
 <![CDATA[
 <p>
-アノテーションは、<code>@Retention(RetentionPolicy.RUNTIME)</code> でアノテートされなければ、リフレクション (たとえば、<code>isAnnotationPresent(...)</code> メソッド) を使用して観測することができません。
+アノテーションに <code>@Retention(RetentionPolicy.RUNTIME)</code> アノテーションが付いていなければ，リフレクション (たとえば， <code>isAnnotationPresent(..)</code> メソッド) を使用して観測することができません。
 </p>
 ]]>
     </Details>
   </BugPattern>
 
   <BugPattern type="DM_EXIT">
-    <ShortDescription>System.exit(...) を呼び出しているメソッド</ShortDescription>
-    <LongDescription>{1} は、System.exit(...) を呼び出しています。これは Java 仮想マシン全体をシャットダウンさせてしまいます。</LongDescription>
+    <ShortDescription>System.exit(..) を呼び出しているメソッド</ShortDescription>
+    <LongDescription>{1} は，System.exit(..) を呼び出しています。これは Java 仮想マシン全体をシャットダウンさせてしまいます。</LongDescription>

```

Comment:
Maybe yes, I think you can keep using `...`, it will not be mixed with varargs like `String...`.